integrationTests/singleShard/block/executingMiniblocks_test.go

```diff
@@ -72,6 +73,7 @@ func TestShardShouldNotProposeAndExecuteTwoBlocksInSameRound(t *testing.T) {
 	mockTestingT := &testing.T{}
 	integrationTests.SyncBlock(mockTestingT, nodes, []int{idxProposer}, nonce)
 
+	runtime.Gosched()
```

Comment:
why is this needed ?

Reply:
this was removed. It was a test to see if this fixed the failing tests on windows